### ISSUE TITLE
Problem loading generated inline partial templates

### ISSUE THREAD

**Issue by alanrsoares** [clkao/angular-livescript-seed#5]
When running on a Windows environment, it does not load the correct partial template for the /about state. It loads the main template instead.
![12-11-2014 1-53-24 p m](https://cloud.githubusercontent.com/assets/273334/5003509/58f440cc-6a73-11e4-8cad-0b2fa91f24db.png)

**Comment by clkao** [comment]
It's likely the paths from `gulp-angular-templatecache` are not turned into '/', can you check the output in `_public/js/app.templates.js`?

**Comment by alanrsoares** [comment]
The generated paths in app.templates.js looks like this:
$templateCache.put("d:/Projects/angular-livescript-seed/app/partials/about.html", [...]
Was it supposed to generate absolute paths?

**Comment by clkao** [comment]
no, it should be relative and same as what templateUrl in `app/app.ls` is referring to.
